src/pages/doi.org/[...doi].tsx

```diff
@@ -426,7 +443,13 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
       context.res.end()
       return { props: {} }
     } else {
-      return { props: { doi, metadata: data } }
+      return {
+        props: {
+          doi,
+          metadata: data,
+          isBot: userAgentFromString(context.req.headers['user-agent']).isBot
```

Comment:
I actually wonder if we can't do this further up and have the isbot passed through, it's probably something we will want to do on other pages. However I am fine with this as fist pass.

Reply:
That would be good, I'll think about where would be a good place to implement it